Homework/Location.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Homework
+{
+    internal class Location
+    {
+        internal string name {  get; set; }
+        internal string region { get; set; }
+        internal string country { get; set; }
+        internal double lat { get; set; }
+        internal double lon { get; set; }
+        internal string tz_id { get; set; }
+        internal long localtime_epoch { get; set; }
+        internal DateTime localtime_time { get; set; }
```

Comment:
Tady bych pouzil string pro jednoduchost a aby ti to fungovalo, protoze ten format odpovedi nema automatickou konverzi na DateTime objekt. Spravna deserializace do tohoto objektu jde udelat (a bezne se to dela), ale je to mozna trochu za hranice tohohle kurzu.
```suggestion
        [JsonPropertyName("localtime")]
        public string LocalTime { get; set; }
```

Ale kdybys chtel abych ti popsal, jak by to slo udelat, tak muzu :)